Control coverage/Feature/SQLServer.md

```diff
@@ -96,6 +97,99 @@ Using the native enterprise directory for authentication ensures that there is a
 
 ___
 
+## Azure_SQLDatabase_AuthZ_Use_AAD_Only
```

Comment:
please help to update that email address would be required if Admin is not set.

Reply:
Adding this into the readme file as it is giving information about BRS